docs/sphinx/quick_start.rst

```diff
@@ -0,0 +1,195 @@
+Quick Start
+*******************************************
+
+**NVIDIA CUDA Quantum**
+
+CUDA Quantum streamlines hybrid application development and promotes productivity and scalability
+in quantum algorithm research. It offers a unified programming model designed for a hybrid
+setting |---| that is, CPUs, GPUs, and QPUs working together. Further enabling application
+acceleration, CUDA Quantum includes language extensions for Python and C++ and a system-level toolchain.
+
+.. 
+    Fix Me: Learn more about CUDA Quantum’s key benefits here [Link to CUDA Quantum Marketing page]
+
+This Quick Start page guides you through getting set up with CUDA Quantum and running your first program.
+If you have already installed and configured CUDA Quantum, we encourage you to move directly to our
+:ref:`Basics Section <cudaq-basics-landing-page>`.
+
+
+Python
+-------
+
+If you want to develop CUDA Quantum applications using Python, install the
+latest stable release of the CUDA Quantum Python API:  
+
+.. code-block:: bash
+
+    pip install cuda-quantum
+
+For further information, see the `CUDA Quantum project <https://pypi.org/project/cuda-quantum/>`_ on PyPI.
+
+We also offer a fully featured CUDA Quantum installation, including all C++ and Python tools, via our
```

Comment:
Consistency in listing languages - I see that now `Python` is called out to be first elsewhere, so perhaps mention `Python and C++` everywhere?